tools/terraform_sync_tool/terraform_sync.py

```diff
@@ -0,0 +1,84 @@
+import json
+import io
+import argparse
+import re
+from google.cloud import bigquery
+
+# Fetch schemas for drifted tables form BQ
+def get_schemas_from_BQ(tables_of_interest, client):
+    table_schemas = []
+    for k in tables_of_interest:
+        table_id = tables_of_interest.get(k).get('table_id')
+        table = client.get_table(table_id)
+        file = io.StringIO("")
+        client.schema_to_json(table.schema, file)
+        # append schema as a dict where key=table_id, value=table_schema
+        schema_bq = dict({table_id: json.loads(file.getvalue())}) 
+        table_schemas.append(schema_bq)
+    return table_schemas
+
+# Identify tables with drifts and add return tables_of_interest 
+def get_drifted_tables(json_file):
+    tables_of_interest = {} # store events(dict) of interest where key=resource_name+resource_key as identifier and value=resource_id                                                                                                    
+    # Opening JSON file
+    with open(json_file) as file:
+        lines = file.readlines()
+        # Scan json lines to store events of interest
+        for line in reversed(lines):                                                                    
+            json_line = json.loads(line)
+            type = json_line.get('type')
+            change = json_line.get('change')   
+            hook = json_line.get('hook') 
+            # When resource_drift event detected, append new dict to tables_of_interest
+            if type == 'resource_drift' and change and change.get('resource') and change.get('resource').get('resource_type') == 'google_bigquery_table':
+                change_resource = change.get('resource')
+                # Use condensed resource_name+resource_key as key of new dict
+                resource_condensed = change_resource.get('resource_name')+change_resource.get('resource_key')
+                # Use resource infomation from resource_drift event in value of new dict
+                resource_from_drift = {'identifier':
+                    {
+                    'resource_name':change_resource.get('resource_name'),
+                    'resource_key':change_resource.get('resource_key')
+                    }}
```

Comment:
You're already storing this information as the key in the dictionary. Why are you also storing it as a value?

Reply:
Updated in https://github.com/GoogleCloudPlatform/bigquery-utils/pull/290/commits/69a2df35cf0d995b0ca94c95a2a5fbc79aeaeb58